src/model/neural_network/neural_network_store.py

```diff
@@ -0,0 +1,62 @@
+import pickle
+from os import makedirs
+from os.path import exists
+from typing import List
+
+from model.neural_network.neural_network import NeuralNetwork
+
+
+class NeuralNetworkStore:
+    """
+    Store and load neural networks from disc.
+    """
+    _DEFAULT_DIRECTORY = 'store'
+
+    @staticmethod
+    def store(neural_networks: List[NeuralNetwork],
+              file_name: str,
+              directory_name: str = _DEFAULT_DIRECTORY) -> None:
+        """
+        Stores neural networks in binary file with .nn extension.
+
+        Args:
+            neural_networks (List[NeuralNetwork]): Neural networks to store in file.
+            file_name (str): Name of file to store neural networks in,
+                will be created if doesn't exist.
+            directory_name (str): Name of directory where file is,
+                will be created if doesn't exist.
+        """
+        if not exists(directory_name):
+            makedirs(directory_name)
+        with open(f'{directory_name}/{file_name}.nn', 'wb') as store:
+            for neural_network in neural_networks:
+                pickle.dump(neural_network, store, pickle.HIGHEST_PROTOCOL)
+
+    @staticmethod
+    def load(file_name: str,
+             directory_name: str = _DEFAULT_DIRECTORY,
+             amount_to_load: int = -1) -> List[NeuralNetwork]:
+        """
+        Loads neural networks from binary file with .nn extension.
+
+        Args:
+            file_name (str): Name of file to load neural networks from
+            directory_name (str): Name of directory where file to load is.
+            amount_to_load (int): Maximal amount of neural networks to load,
+                if there is no enough neural networks in file,
+                everything will be loaded without any exception,
+                same for negative value.
+        Raises:
+            FileNotFoundError: If file or directory name doesn't exist.
```

Comment:
Names always exist as strings, it's files and directories themselves that don't always exist.